src/www/ui/scripts/change-license-view.js

```diff
@@ -94,8 +94,7 @@ function scheduleBulkScan() {
   scheduleBulkScanCommon($('#bulkIdResult'), reloadClearingTable);
 }
 
-function cleanText() {
-  var $textField = $('#bulkRefText');
+function cleanText($textField) {
```

Comment:
I believe starting the variable with a `$` is a typo in JS. Can you please remove the `$` from `textField`?

Reply:
And unfortunately, the function is duplicated in following place as well. Can you please update the same to have the same signature?

https://github.com/fossology/fossology/blob/4ce46a3ed1490dce672c3b02fea8aff930017f36/src/www/ui/scripts/change-license-browse.js#L136